validator/src/main/java/com/amazon/aoc/fileconfigs/PredefinedExpectedTemplate.java

```diff
@@ -168,6 +173,60 @@ public enum PredefinedExpectedTemplate implements FileConfig {
   PYTHON_K8S_CLIENT_CALL_LOG("/expected-data-template/python/k8s/client-call-log.mustache"),
   PYTHON_K8S_CLIENT_CALL_METRIC("/expected-data-template/python/k8s/client-call-metric.mustache"),
   PYTHON_K8S_CLIENT_CALL_TRACE("/expected-data-template/python/k8s/client-call-trace.mustache"),
+
+  /** DotNet EKS Test Case Validations */
+  DOTNET_EKS_OUTGOING_HTTP_CALL_LOG("/expected-data-template/dotnet/eks/outgoing-http-call-log.mustache"),
+  DOTNET_EKS_OUTGOING_HTTP_CALL_METRIC("/expected-data-template/dotnet/eks/outgoing-http-call-metric.mustache"),
+  DOTNET_EKS_OUTGOING_HTTP_CALL_TRACE("/expected-data-template/dotnet/eks/outgoing-http-call-trace.mustache"),
+
+  DOTNET_EKS_AWS_SDK_CALL_LOG("/expected-data-template/dotnet/eks/aws-sdk-call-log.mustache"),
+  DOTNET_EKS_AWS_SDK_CALL_METRIC("/expected-data-template/dotnet/eks/aws-sdk-call-metric.mustache"),
+  DOTNET_EKS_AWS_SDK_CALL_TRACE("/expected-data-template/dotnet/eks/aws-sdk-call-trace.mustache"),
+
+  DOTNET_EKS_REMOTE_SERVICE_LOG("/expected-data-template/dotnet/eks/remote-service-log.mustache"),
+  DOTNET_EKS_REMOTE_SERVICE_METRIC("/expected-data-template/dotnet/eks/remote-service-metric.mustache"),
+  DOTNET_EKS_REMOTE_SERVICE_TRACE("/expected-data-template/dotnet/eks/remote-service-trace.mustache"),
+
+  DOTNET_EKS_CLIENT_CALL_LOG("/expected-data-template/dotnet/eks/client-call-log.mustache"),
+  DOTNET_EKS_CLIENT_CALL_METRIC("/expected-data-template/dotnet/eks/client-call-metric.mustache"),
+  DOTNET_EKS_CLIENT_CALL_TRACE("/expected-data-template/dotnet/eks/client-call-trace.mustache"),
```

Comment:
Remove all of these and the files, they shouldn't be part of the scope of this PR.

Reply:
Removed